packages/test/test-utils/src/index.ts

```diff
@@ -25,6 +25,7 @@ export {
 export {
 	createSummarizer,
 	createSummarizerFromFactory,
+    createSummarizerWithContainer,
```

Comment:
whitespace nit:
```suggestion
        createSummarizerWithContainer,
```

Reply:
I think the sin of tabs were committed in this file.